caf: modules: ble_adv: Power down on disconnection with reason 0x15 (#13987)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

  * Updated the dependencies of the :kconfig:option:`CONFIG_CAF_BLE_USE_LLPM` Kconfig option.
    The option can be enabled even when the Bluetooth controller is not enabled as part of the application that uses :ref:`caf_ble_state`.

> * :c:struct:`ble_peer_event`:

Reviewer: We do not usually refer to the structures on this level of the changelog. Is this related to a specific CAF module?

  * Added :c:member:`ble_peer_event.reason` to inform about reason code related to state of the Bluetooth LE peer.
    The field is used to propagate information about error code related to a connection establishment failure and disconnection reason.

* :ref:`caf_ble_adv`:

>   * Added :kconfig:option:`CONFIG_CAF_BLE_ADV_POWER_DOWN_ON_DISCONNECTION_REASON_0X15` Kconfig option.

Reviewer: ```suggestion
  * Added the :kconfig:option:`CONFIG_CAF_BLE_ADV_POWER_DOWN_ON_DISCONNECTION_REASON_0X15` Kconfig option.
```

>     The option can be used to force system power down when a bonded peer disconnects with reason ``0x15`` (Remote Device Terminated due to Power Off).

Reviewer: ```suggestion
    You can use this option to force system power down when a bonded peer disconnects with reason ``0x15`` (Remote Device Terminated due to Power Off).
```
